ansible/roles/prometheus/defaults/main.yml

```diff
@@ -43,5 +43,41 @@ prometheus_configuration:
       static_configs:
         - targets: ['localhost:9100']
 
+    - job_name: blackbox-ssh
+      metrics_path: /probe
+      params:
+        module: [ssh_banner]
+
+      static_configs:
+        - targets:
+            - lovelace.box.pydis.wtf
+
+      relabel_configs:
+        # Ensure that the SSH port is included explicitly
+        - source_labels: [__address__]
+          regex: (.*?)(:.*)?
+          replacement: ${1}:22
+          target_label: __param_target
+
+        - source_labels: [__param_target]
+          target_label: instance
+
+        - target_label: __address__
+          replacement: localhost:9115
+
+    - job_name: blackbox-http
+      metrics_path: /probe
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+            - https://pydis.wtf/
```

Comment:
We should also add https://cloud.native.is.fun.and.easy.pydis.wtf to the list

Reply:
Granted